### ISSUE TITLE
Unable to reproduce 75% accuracy on SID task with wav2vec2

### ISSUE THREAD

**Issue by david-wei-01001** [s3prl/s3prl#582]
Dear s3prl manager,
with the default setting of SID task (as in voxceleb1 config.yaml), I am unable to get the reported 75.18% accuracy with model wav2vec 2.0 Base.
The command I used to run is 
```bash
python3 run_downstream.py -m train -n w2v2Exp -u wav2vec2 -d voxceleb1
```
may I please know the hyperparameter setting?
Is it 200000 steps with learning rate 1.0e-4?

Thank you very much!

**Comment by david-wei-01001** [comment]
solved, wit h lr 1e-1
